addons/web_editor/static/src/js/wysiwyg/widgets/link_popover_widget.js

```diff
@@ -85,28 +93,32 @@ const LinkPopoverWidget = Widget.extend({
             container: this.options.wysiwyg.odooEditor.document.body,
         })
         .on('show.bs.popover.link_popover', () => {
-            this.options.wysiwyg.odooEditor.observerUnactive('show.bs.popover');
             this._loadAsyncLinkPreview();
             popoverShown = true;
         })
-        .on('inserted.bs.popover', () => {
-            this.options.wysiwyg.odooEditor.observerActive('show.bs.popover');
-        })
         .on('hide.bs.popover.link_popover', () => {
-            this.options.wysiwyg.odooEditor.observerUnactive('hide.bs.popover');
             popoverShown = false;
         })
         .on('hidden.bs.popover.link_popover', () => {
-            this.options.wysiwyg.odooEditor.observerActive('hide.bs.popover');
             for (const tooltip of tooltips) {
                 tooltip.hide();
             }
         })
         .on('inserted.bs.popover.link_popover', () => {
             this.$target.data('bs.popover').tip.classList.add('o_edit_menu_popover');
-        })
-        .popover('show');
+        }).popover('show');
+        this.options.wysiwyg.odooEditor.observerActive();
 
+        // Init popover inner tooltips (no need of observer unactive since out
+        // of the editable area).
```

Comment:
_Question:_
Surprising :o 
Only true since https://github.com/odoo/odoo/commit/161c5fc8e742294e8d85c889b4bf8d3d8cd484f5#diff-44e2f4c9175434219f95516b77671359e8299bde247501bcd945c5e68c59c1bbR72 I guess? But it seems older than the code introducing the observer shenanigans on all tooltip/popover events so maybe not.

Reply:
> Only true since

Probably indeed. TBH I just tested that when moving that code around to be sure, but I thought opening in the body was the default of bootstrap. Some luck then I guess. At worst if this changes, my test should catch it.